src/backend/impress/settings.py

```diff
@@ -938,6 +957,11 @@ class Test(Base):
     # Tests are raising warnings because the /data/static directory does not exist
     STATIC_ROOT = None
 
+    # Setup indexer configuration to make test working on the CI.
+    SEARCH_INDEXER_SECRET = "ThisIsAKeyForTest"  # noqa
+    SEARCH_INDEXER_URL = "http://localhost:8081/api/v1.0/documents/index/"
+    SEARCH_INDEXER_QUERY_URL = "http://localhost:8081/api/v1.0/documents/search/"
+
```

Comment:
I'm not sure to understand. 

My two cents:
 - you should only set the settings in the tests which requires it
 - there must be a way to disable the indexation, because not every instance of docs will have an index, and it must be the default case.
 
 => I would remove those settings, determine which settings enables the indexation (I think SEARCH_INDEXER_CLASS should be None by default, and if None, then the document must not be indexed). Add new tests on the documents save, with SEARCH_INDEXER_CLASS not None, to check the indexation task etc.